io.github.wojciech_graj.OrbVis.yml

```
@@ -0,0 +1,42 @@
+app-id: io.github.wojciech_graj.OrbVis
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+platform-extensions:
+  - org.freedesktop.Platform.GL.default
+sdk: org.freedesktop.Sdk
+command: orbvis
+modules:
+  - name: cglm
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://github.com/recp/cglm/archive/refs/tags/v0.8.6.tar.gz
+        sha256: 6f6c9f7ed150b6c691774cc2750a79f363f0aa9acadade85e5ff8bb5b36eac28
+  - name: orbvis
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://github.com/wojciech-graj/OrbVis/releases/download/v0.2.2/orbvis-0.2.2-src-minimal.zip
+        sha256: c4bc189f3e22a700844c683666f06a170a7da265297bbc18b08db9f77c1bd5de
+        strip-components: 0
+  - name: metainfo
+    buildsystem: simple
+    build-commands:
+      - install -D -m755 io.github.wojciech_graj.OrbVis.appdata.xml /app/share/appdata/io.github.wojciech_graj.OrbVis.appdata.xml
+      - install -D -m755 128x128/io.github.wojciech_graj.OrbVis.png  /app/share/icons/hicolor/128x128/apps/io.github.wojciech_graj.OrbVis.png
+      - install -D -m755 64x64/io.github.wojciech_graj.OrbVis.png /app/share/icons/hicolor/64x64/apps/io.github.wojciech_graj.OrbVis.png
+      - install -D -m755 io.github.wojciech_graj.OrbVis.desktop /app/share/applications/io.github.wojciech_graj.OrbVis.desktop
+    sources:
+      - type: file
+        path: io.github.wojciech_graj.OrbVis.appdata.xml
+      - type: file
+        path: io.github.wojciech_graj.OrbVis.desktop
+      - type: dir
+        path: icons
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

considering it supports wayland. Otherwise the wayland socket shall be removed.